src/assets/js/main.js

```diff
@@ -20,15 +20,15 @@ $(function () {
   prettyPrint();
 });
 
-// TODO(chalin): Copied (& tweaked) from site-www, consider moving into site-shared
 function adjustToc() {
   // Adjustments to the jekyll-toc TOC.
 
   var tocId = '#site-toc--side';
-  var tocWrapper = $(tocId);
-  $(tocWrapper).find('.site-toc--button__page-top').click(function () {
-    $('html, body').animate({ scrollTop: 0 }, 'fast');
-  })
+  // Uncomment to enable 'page top' button
```

Comment:
Do you want to delete this, or is intended to be brought back in at some point?

Reply:
@sfshaza2 Have we ever used a back to page top button on the site? It's interesting to me that the button is hidden.